lambdaclient/runtime/src/main/java/io/quarkus/amazon/lambda/client/runtime/LambdaConfig.java

```diff
@@ -0,0 +1,91 @@
+package io.quarkus.amazon.lambda.client.runtime;
+
+import java.util.Optional;
+
+import io.quarkus.amazon.common.runtime.AsyncHttpClientConfig;
+import io.quarkus.amazon.common.runtime.AwsConfig;
+import io.quarkus.amazon.common.runtime.SdkConfig;
+import io.quarkus.amazon.common.runtime.SyncHttpClientConfig;
+import io.quarkus.runtime.annotations.ConfigDocSection;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithDefault;
+import io.smallrye.config.WithParentName;
+
+@ConfigMapping(prefix = "quarkus.lambda")
+@ConfigRoot(phase = ConfigPhase.RUN_TIME)
+public interface LambdaConfig {
+
+    /**
+     * Enable using the accelerate endpoint when accessing Lambda.
+     *
+     * <p>
+     * Accelerate endpoints allow faster transfer of objects by using Amazon CloudFront's globally
+     * distributed edge locations.
+     */
+    @WithDefault("false")
+    boolean accelerateMode();
```

Comment:
remove non necessary/used property for this service

Reply:
Addressed